src/frontend/src/states/ApiState.tsx

```diff
@@ -206,5 +204,11 @@ export function apiUrl(path: ApiPaths, pk?: any): string {
     _url += `${pk}/`;
   }
 
+  if (_url && data) {
+    for (const key in data) {
+      _url = _url.replace(`:${key}`, data[key]);
```

Comment:
To fix the issue you would need something like this, that was actually what I was trying to explain in my previous comment.
```suggestion
      _url = _url.replace(`:${key}`, `${data[key]}`);
```